taskhawk/backends/aws.py

```diff
@@ -173,6 +173,10 @@ def process_message(self, queue_message) -> None:
     def delete_message(self, queue_message) -> None:
         queue_message.delete()
 
+    def nack_message(self, queue_message) -> None:
+        # should operate like a nack https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-visibility-timeout.html#terminating-message-visibility-timeout
```

Comment:
aws doesnt say NACK in particular but I think what they describe here is the same?

Reply:
Yep that's nack.